### PR TITLE
Cool columns of tutorials sorted by experience level.

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

File: source/tutorials/index.rst [hunk 1]
```diff
@@ -3,9 +3,56 @@
 Tutorials
 #########
 
-Explore our tutorials to discover what you can do with |CL|!
+Explore our tutorials to discover what you can do with |CL|! Now conveniently
+sorted by experience level!
```
Comment (mvincerx) [inline source/tutorials/index.rst:7]: Hey @intelkevinputnam. Let's discuss a standard word in place of "experience level"--based on our emails about this. 
fyi: @bktan8, @krisdale

Whatever we decide on for "experience level," that phrase can be also be a hyperlink to a new document
where we explain the rating levels.

File: source/tutorials/index.rst [hunk 2]
```diff
@@ -3,9 +3,56 @@
...
+      .. rst-class:: colh3
+
+      Moderate
+
+      -  :ref:`hadoop`
+      -  :ref:`spark`
+      -  :ref:`broadcom`
+      -  :ref:`fmv`
+      -  :ref:`kubernetes`
+      -  :ref:`kubernetes-bp`
```
Comment (mvincerx) [inline source/tutorials/index.rst:37]: Will you please move `kubernetes` to High rating? Thanks. 
Docs team agreed on this in our meeting.

Comment (mvincerx) [reply source/tutorials/index.rst:37]: After this tutorial is moved to High, I'm ready to merge.

File: source/tutorials/index.rst (hunk 1 above)
Comment (bktan8) [reply source/tutorials/index.rst:7]: Low, Moderate, and High are not descriptions for "experience level".  If we're going with levels, they should be "Beginner, Immediate, Advanced"?

Comment (mvincerx) [reply source/tutorials/index.rst:7]: Easy, Moderate, Difficult.  Nice. Thanks, all.

**Comment by intelkevinputnam** [comment]
@mvincerx Should be good to go now. I would recommend, however, that we revisit some of the Tutorial names that are now newly on the top level. We need to remove "on Clear Linux" and generally make them shorter:

1. wordpress.rst
2. hpc.rst
3. lamp-server.rst

Are the ones I'm seeing right now.

**Comment by mvincerx** [comment]
> @mvincerx Should be good to go now. I would recommend, however, that we revisit some of the Tutorial names that are now newly on the top level. We need to remove "on Clear Linux" and generally make them shorter:
> 
> 1. wordpress.rst
> 2. hpc.rst
> 3. lamp-server.rst
> 
Agreed on tutorial names being shorter. Will you please file an issue for this?

Merging this PR now.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**